rest.rst

```diff
@@ -285,68 +284,6 @@ Selecting Devices to Test On
 
 To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``. Or you can also set the devices the moment you create a testrun using the `Create a new testrun`_ endpoint.
 
-Start an existing testrun
```

Comment:
Yes leave it but we should describe it a little bit more. If it's unclear for us, 10x it will be unclear for the client.

Also the intro section has to be extended where we say the following:

You can use the API to:

Upload an app
Upload an app and automatically start that testrun
Start an existing testrun
By now our API can do way more :)


Martin

Sent from my iPhone

> On May 19, 2015, at 1:53 PM, Łukasz Fryc <notifications@github.com> wrote:
> 
> In rest.rst:
> 
> > @@ -285,68 +284,6 @@ Selecting Devices to Test On
> >  
> >  To select devices to test on, go to testmunk.com and navigate to ``Account Settings`` > ``REST API``. Or you can also set the devices the moment you create a testrun using the `Create a new testrun`_ endpoint.
> >  
> > -Start an existing testrun
> @mposchen Actually, it's possible to start an existing testrun. I've just found how. First you need to create new testrun and set autoStart=false: 
> 
> 
> And then, you need to have the "id" (from previous call) and you can use /run to start it.
> 
> There are not many use cases for this. Nevertheless, we can leave it. @mposchen How do you think?
> 
> —
> Reply to this email directly or view it on GitHub.
>